core/src/main/java/tech/tablesaw/table/Relation.java

```diff
@@ -240,19 +240,55 @@ public Table structure() {
     return structure;
   }
 
-  public String summary() {
-    StringBuilder builder = new StringBuilder();
-    builder
-        .append(System.lineSeparator())
-        .append("Table summary for: ")
-        .append(name())
-        .append(System.lineSeparator());
-    for (Column<?> column : columns()) {
-      builder.append(column.summary().print());
-      builder.append(System.lineSeparator());
+  /*public Table summary() {
```

Comment:
We prefer to delete code rather than comment it out because it can confuse other people who come later and wonder why it's here and commented out

Reply:
@emilianbold My general view is that it is better to have consistent API's across the codebase. The summary() API on column returns a Table so it feels more natural to me to do the same here. It is easy enough to get the String representation from that by calling the existing print() methods.
If the preference is to rename all the summary() methods to describe(), including on the columns, I'd support that too.
Personally I don't find the current String representation of summary() useful, as it seems to lazily dump column stats individually which is somewhat hard to read.
I'm happy to defer to the maintainers here though, @benmccann,  @lwhite1 let me know which you prefer. 

@benmccann Agree, was not my intent to leave in commented out code. I'll fix, apologies.

Reply:
You have to think a bit about backwards compatibility too. Existing code might be out there that assumes the return type is string. That code will not compile anymore. That was my angle against changing the return type so easily.

Reply:
We've never been particularly strict about backwards compatibility since the library is pre-1.0. And I imagine in this case that if you do something like `System.out.println(table.summary());` that it won't really matter that the return type changed

My main question was whether we'd also want a row in the summary table that prints the column type since the old output included and it could be useful in some circumstances

Reply:
> We've never been particularly strict about backwards compatibility since the library is pre-1.0.

Good to know. This allows more flexibility in other parts too.